.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+version: 2.1
+executors:
+  docker-executor:
+    docker:
+      - image: circleci/openjdk:17-buster-node-browsers-legacy
+
+jobs:
+  build_and_push_tag_image:
```

Comment:
@somanath21 Are we following consistent naming here? We are using hyphen ('-') separated in line 3 and underscore ('_') separated here.

Reply:
This is Job name and same will be called in workflow. No naming convention here in job names